lte/gateway/c/session_manager/SessionState.cpp

```diff
@@ -219,7 +219,10 @@ SessionState::SessionState(
       config_(cfg),
       current_version_(0),
       rtx_counter_(0),
-      static_rules_(rule_store) {}
+      static_rules_(rule_store),
+      credit_map_(4, &ccHash, &ccEqual),
```

Comment:
looks like there's a GCC warning regarding the order here. It should be the same order as how it is declared in the class

Reply:
Fixed it.